[2023-12-25T10:22:39.646+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: ETL.Data_Cleaning manual__2023-12-25T10:17:35.996651+00:00 [queued]>
[2023-12-25T10:22:39.650+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: ETL.Data_Cleaning manual__2023-12-25T10:17:35.996651+00:00 [queued]>
[2023-12-25T10:22:39.651+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2023-12-25T10:22:39.651+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2023-12-25T10:22:39.651+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2023-12-25T10:22:39.658+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): Data_Cleaning> on 2023-12-25 10:17:35.996651+00:00
[2023-12-25T10:22:39.663+0000] {standard_task_runner.py:52} INFO - Started process 934 to run task
[2023-12-25T10:22:39.666+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL', 'Data_Cleaning', 'manual__2023-12-25T10:17:35.996651+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_saepul_hilal_DAG.py', '--cfg-path', '/tmp/tmpalockljg', '--error-file', '/tmp/tmpqu0cyxut']
[2023-12-25T10:22:39.667+0000] {standard_task_runner.py:80} INFO - Job 11: Subtask Data_Cleaning
[2023-12-25T10:22:39.700+0000] {task_command.py:371} INFO - Running <TaskInstance: ETL.Data_Cleaning manual__2023-12-25T10:17:35.996651+00:00 [running]> on host 0df44d59871b
[2023-12-25T10:22:39.738+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=saepul
AIRFLOW_CTX_DAG_ID=ETL
AIRFLOW_CTX_TASK_ID=Data_Cleaning
AIRFLOW_CTX_EXECUTION_DATE=2023-12-25T10:17:35.996651+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-12-25T10:17:35.996651+00:00
[2023-12-25T10:22:39.840+0000] {python.py:173} INFO - Done. Returned value was: None
[2023-12-25T10:22:39.848+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=ETL, task_id=Data_Cleaning, execution_date=20231225T101735, start_date=20231225T102239, end_date=20231225T102239
[2023-12-25T10:22:39.883+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-25T10:22:39.903+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
